src/server/server.cc

```diff
@@ -846,13 +846,18 @@ void Server::GetRocksDBInfo(std::string *info) {
   db->GetAggregatedIntProperty("rocksdb.num-live-versions", &num_live_versions);
 
   string_stream << "# RocksDB\r\n";
+
+  uint64_t block_cache_usage = 0;
+  // All column families share the same block cache, so it's good to count a single one.
+  auto cf_handle = storage->GetCFHandle(engine::kSubkeyColumnFamilyName);
+  db->GetIntProperty(cf_handle, "rocksdb.block-cache-usage", &block_cache_usage);
+  string_stream << "block_cache_usage:" << block_cache_usage << "\r\n";
+
   for (const auto &cf_handle : *storage->GetCFHandles()) {
```

Comment:
Hi, sonar Cloud telling us we declaration shadows a local variable is not clear. 
Can you change the variable name?
https://sonarcloud.io/project/issues?open=AY7Xa5ShOhsi8ezKPlra&id=apache_kvrocks